toptal/picasso
# fix(tooltip): fix compact size variant for tooltip component

### Description

This is a small fix related to the feature introduced on #1532.

**Design links:**
- [Slider](https://share.goabstract.com/cb4d4d06-011a-4514-8542-ba2d0a08ce9d?collectionLayerId=9f25bdbd-f87f-4fd2-bfe2-61b456d0daeb&mode=design)
- [Tooltip](https://share.goabstract.com/95efe1da-0129-4a8a-a2f9-d529ef8b6d09?mode=design&sha=8f2b4e14203d5ae7d647aa7a1e29ff7db037e921)

### How to test

- Verify `Tooltip` [example](https://picasso.toptal.net/fix-compact-tooltip-size/?path=/story/overlays-folder--tooltip#compact) with `compact` prop
- Verify `Slider` displayed with a [compact tooltip](https://picasso.toptal.net/fix-compact-tooltip-size/?path=/story/components-folder--slider#tooltip)

### Screenshots

| `Tooltip` with `compact` prop                                 | Slider with `compact` prop                                  |
| --------------------------------------- | --------------------------------------- |
| **Before:** <br> <img width="114" alt="Screen Shot 2020-09-01 at 10 12 37" src="https://user-images.githubusercontent.com/4870595/91855352-7053ff00-ec3b-11ea-869b-552a6dadd426.png"> <br> **After:** <br> <img width="139" alt="Screen Shot 2020-09-14 at 11 28 58" src="https://user-images.githubusercontent.com/4870595/93126965-77e7bf00-f6a3-11ea-901f-f8644b8ff575.png"> | **Before:** <br> <img width="540" alt="Screen Shot 2020-09-01 at 10 24 59" src="https://user-images.githubusercontent.com/4870595/91856546-15bba280-ec3d-11ea-8d9a-cee40fa7e1bc.png"> <br> **After:** <br> <img width="493" alt="Screen Shot 2020-09-14 at 11 30 15" src="https://user-images.githubusercontent.com/4870595/93127082-a796c700-f6a3-11ea-8bf2-bb3722e2bf1b.png"> |
### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
